pyanaconda/geoloc.py

```diff
@@ -217,7 +219,8 @@ def _log_geolocation_status(self, geolocation_enabled, options_override):
                 log.info("Geolocation is disabled for image or directory installation.")
             elif flags.automatedInstall:
                 log.info("Geolocation is disabled due to automated kickstart based installation.")
-            if not flags.cmdline.getbool('geoloc', True):
+            if not kernel_arguments.get('geoloc', default=True, interpret_booleans=True) or \
+                    "nogeoloc" in kernel_arguments:
```

Comment:
Btw. I don't think that we should parse the kernel argument `geoloc` here. It is our option, so it should be handled during parsing kernel and cmdline arguments and merged with `opt.geoloc`. This is just wrong, but out of scope of this pull request.